src/features/pages/partners/deriv-prime/hero/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import { hero_wrapper, container, hero_img } from './styles.module.scss'
+import { Localize, localize } from 'components/localization'
+import LinkButton from 'features/components/atoms/link-button'
+import Typography from 'features/components/atoms/typography'
+import Flex from 'features/components/atoms/flex-box'
+
+const Hero = () => {
+    return (
+        <Flex.Box
+            bgcolor="black"
+            container="fixed"
+            as="section"
+            align="center"
+            pt="39x"
+            className={hero_wrapper}
+        >
+            <div className={hero_img}>
+                <StaticImage
+                    src="../../../../../images/common/deriv-prime/hero.png"
```

Comment:
better to import and use as variable instead of writing `../../../../../images/common/deriv-prime/hero.png` 
please, do like this `src={Hero}`

Reply:
Gatsby StaticImage only support relative path and URL.

Reply:
got it, my bad, thanks